libs/shared/iam/feature/src/lib/use-user-role/use-user-role.ts

```diff
@@ -31,6 +38,7 @@ export function useUserRole() {
     roles,
     loading,
     isQoveryAdminUser,
+    isAdminOrOwnerOfCompany,
```

Comment:
I think it's maybe better to return `role` props and add the condition directly in the feature side, wdty?

Reply:
Yeah, why not! Let me update that 🙂